inception/inception-api-annotation-experimental/src/main/ts/client/messages/request/relation/CreateRelationRequest.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export class CreateRelationRequest
+{
+    clientName : string;
+    userName : string;
```

Comment:
Because if a client, e.g. christoph, works in parallel (e.g. splitscreen) on an annotation document for user "christoph" and "max", the server needs to know for which annotation document the relation / span / etc was requested. Therefore, the requests needs to cover the clientname who has sent the request (in order to send it properly back to the correct client), but also the username of the annotation document, for which a request was made (because it may not be the annotation document of the client who made the request)

Reply:
I can understand why you would have an "annotator name" on the viewport. I could also understand having an "annotator name" (what you call userName) - but note that at least for now we do not have a case where one user  would be changing the annotations of another user - a user can usually only edit the users own annotation document. But if the client name is the name of the user currently logged in, then it should come from the login session and not from the client - otherwise you could pose as another user, couldn't you?